packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/TreeSitterScopeHandler/TreeSitterScopeHandler.ts

```diff
@@ -33,38 +35,49 @@ export class TreeSitterScopeHandler extends BaseTreeSitterScopeHandler {
   protected matchToScope(
```

Comment:
this function turned out pretty ugly.  I wonder if I should have another version of `getRelatedRange` that just returns a `range` like it used to, as don't care about `allowMultiple` for any of the related ranges other than `.iterationScope`